guides/v2.2/install-gde/prereq/security.md

```diff
@@ -1 +1,66 @@
+
+{:.bs-callout .bs-callout-info}
+  Magento has no recommendation about using SELinux; you can use it for enhanced security if you wish. If you use SELinux, you must configure it properly or the Magento application will function unpredictably. If you choose to use SELinux, consult a resource like [the CentOS wiki](http://wiki.centos.org/HowTos/SELinux){:target="_blank"} to set up rules to enable communication.
+
+### Suggestion for installing the Magento software with Apache
+
+If you choose to enable SELinux, you might have issues running the installer unless you change the *security context* of some directories as follows:
+
+	chcon -R --type httpd_sys_rw_content_t <your Magento install dir>/app/etc
+	chcon -R --type httpd_sys_rw_content_t <your Magento install dir>/var
+	chcon -R --type httpd_sys_rw_content_t <your Magento install dir>/pub/media
+	chcon -R --type httpd_sys_rw_content_t <your Magento install dir>/pub/static
+  chcon -R --type httpd_sys_rw_content_t <your Magento install dir>/generated
+
+The preceding commands work only with the Apache web server. Because of the variety of configurations and security requirements, we don't guarantee these commands work in all situations. For more information, see:
+
+*	[man page](http://linux.die.net/man/8/httpd_selinux){:target="_blank"}
+*	[serverlab](http://www.serverlab.ca/tutorials/linux/web-servers-linux/configuring-selinux-policies-for-apache-web-servers/){:target="_blank"}
+
+### Enable inter-server communication
+
+If Apache and the database server are on the same host, you can skip this section and continue with [Opening Ports In Your Firewall](#install-iptables).
+
+To enable Apache to initiate a connection to another host with SELinux enabled:
+
+1.	To determine if SELinux is enabled, use the following command:
+
+		getenforce
+
+	`Enforcing` displays to confirm that SELinux is running.
+
+2.	Enter one of the following commands:
+
+	CentOS: `setsebool -P httpd_can_network_connect=1`
+
+	Ubuntu: `setsebool -P apache2_can_network_connect=1`
+
+## Opening Ports In Your Firewall {#install-iptables}
+
+Depending on your security requirements, you might find it necessary to open port 80 and other ports in your firewall. Because of the sensitive nature of networking security, Magento strongly recommends you consult with your IT department before proceeding. Following are some suggested references:
+
+*	Ubuntu: [Ubuntu documentation page](https://help.ubuntu.com/community/IptablesHowTo){:target="_blank"}.
+*	CentOS: [CentOS how-to](http://wiki.centos.org/HowTos/Network/IPTables){:target="_blank"} and [CentOS reference page](http://www.centos.org/docs/4/4.5/Security_Guide/s1-firewall-ipt-basic.html){:target="_blank"}.
+
+#### Related topics:
```

Comment:
Please delete the **Related topics:** section. We don't really need these links.